### PR TITLE
Sul23 160 sprinkles option for horizontal cards

### PR DESCRIPTION
**Pull request by mdyoung3**
# READY
- (Edit the above to reflect status)

# Summary
- Adds same behaviors from card in teaser to single-layout card. (https://github.com/SU-SWS/ace-sulgryphon/blob/feaedbab8796ac0a53412e3f003a4d9f67ae1580/drush/sites/stanfordlagunita.site.yml)
- Additional ddev configs (prevents ddev from editing settings.php)

# Review By (Date)
- When does this need to be reviewed by?

# Urgency
- Med

# Steps to Test

1. Do this
1. Then this
2. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: docroot/profiles/custom/sul_profile/modules/sul_helper/src/Plugin/paragraphs/Behavior/CardBehaviors.php [hunk 1]
```diff
@@ -54,6 +54,34 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
         'cta_button' => $this->t('CTA'),
       ],
     ];
+    $element['background'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Background Color'),
+      '#description' => $this->t('Choose a background for the component.'),
+      '#empty_option' => $this->t('- Change the background -'),
+      '#default_value' => $paragraph->getBehaviorSetting('sul_teaser_styles', 'background'),
```
Comment (pookmish) [inline docroot/profiles/custom/sul_profile/modules/sul_helper/src/Plugin/paragraphs/Behavior/CardBehaviors.php]: This must match the ID of this plugin all the time. [see the id](https://github.com/SU-SWS/ace-sulgryphon/pull/29/files#diff-dd83ebd19befcb103b16eec970b6303b9e4443d7d58b88dba133e0cd7b173968R16)
```suggestion
      '#default_value' => $paragraph->getBehaviorSetting('sul_card_styles', 'background'),
```

File: docroot/profiles/custom/sul_profile/modules/sul_helper/src/Plugin/paragraphs/Behavior/CardBehaviors.php [hunk 2]
```diff
@@ -54,6 +54,34 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
...
+      '#options' => [
+        'black' => $this->t('Black'),
+      ],
+    ];
+    $element['background_sprinkles'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Background Sprinkles'),
+      '#description' => $this->t('Choose the position of the "sprinkles".'),
+      '#empty_option' => $this->t('- Change the position -'),
+      '#default_value' => $paragraph->getBehaviorSetting('sul_teaser_styles', 'background_sprinkles'),
```
Comment (pookmish) [inline docroot/profiles/custom/sul_profile/modules/sul_helper/src/Plugin/paragraphs/Behavior/CardBehaviors.php]:
```suggestion
      '#default_value' => $paragraph->getBehaviorSetting('sul_card_styles', 'background_sprinkles'),
```

File: docroot/profiles/custom/sul_profile/modules/sul_helper/src/Plugin/paragraphs/Behavior/CardBehaviors.php [hunk 3]
```diff
@@ -54,6 +54,34 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
...
+      '#options' => [
+        'top_left' => $this->t('Top Left'),
+        'top_right' => $this->t('Top Right'),
+        'bottom_left' => $this->t('Bottom Left'),
+        'bottom_right' => $this->t('Bottom Right'),
+      ],
+      '#states' => [
+        'invisible' => [
+          'select[name="behavior_plugins[sul_teaser_styles][background]"]' => ['value' => ''],
```
Comment (pookmish) [inline docroot/profiles/custom/sul_profile/modules/sul_helper/src/Plugin/paragraphs/Behavior/CardBehaviors.php]: I'm not sure if this is the reason why the state doesn't work. You'll need to review the functionality.
```suggestion
          'select[name="behavior_plugins[sul_card_styles][background]"]' => ['value' => ''],
```

**Review by pookmish — CHANGES_REQUESTED** [review]
This needs some changes before it will even function correctly

**Comment by mdyoung3** [comment]
@pookmish Sorry. I should have caught that. Updating.

**Comment by mdyoung3** [comment]
@pookmish I removed the background and update the sprinkles option. 

Ready for review.

**Review by pookmish — APPROVED** [review]
Thanks

**Pull request merged**